bin-dev-scripts/standard-version-release.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+npm install
+npm test
+git diff --quiet
+
+BRANCH=$(git branch | grep \* | cut -d ' ' -f2-)
+
+if [ $BRANCH == 'master' ]
+then
+  npx standard-version
+else
+  npx standard-version --prerelease $BRANCH
+fi
+
+unset BRANCH
+git push --follow-tags origin
```

Comment:
Mention which branch you push to

Reply:
And please put this file in /bin

Reply:
I didn't want the dev-time scripts to be part of the bundled package. So, we can put it in the bin folder and ensure that these files are not bundled into the tarball.